### PR TITLE
coreaction: init at 4.2.0

### PR DESCRIPTION
**Pull request by dan4ik605743**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -6361,6 +6361,8 @@ in
 
   libscrypt = callPackage ../development/libraries/libscrypt { };
 
+  libcsys = libsForQt5.callPackage ../development/libraries/libcsys { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:6364]: You are missing libcprime.

Comment (dan4ik605743) [reply pkgs/top-level/all-packages.nix:6364]: > You are missing libcprime.

https://github.com/NixOS/nixpkgs/pull/130737

Comment (dan4ik605743) [reply pkgs/top-level/all-packages.nix:6364]: libcprime There, I understand that you start these 2 PR at the same time and they will fall into the channel at one time. And everything will work

**Comment by SuperSandro2000** [comment]
@dan4ik605743 please resolve the merge conflict.

**Comment by dan4ik605743** [comment]
hmm

**Comment by dan4ik605743** [comment]
> @dan4ik605743 please resolve the merge conflict.

@SuperSandro2000 I rearrange PR and make Pull. And now I do not understand anything.

**Comment by dan4ik605743** [comment]
shit...

**Comment by dan4ik605743** [comment]
@SuperSandro2000 
image
I want to make Squash well, no way it does not work ... New PR Create chtoli ..

**Comment by dan4ik605743** [comment]
@SuperSandro2000 OK. What kind of porridge. I rearrange PR with the new Branch

**Pull request closed without merging**